### ISSUE TITLE
env: node\r: No such file or directory

### ISSUE THREAD

**Issue by beeman** [SoCreate/angular-playground#3]
I'm getting the error `env: node\r: No such file or directory` when trying to run angular-playground in a freshly installed angular-cli app.

To reproduce (using `angular-cli: 1.0.0-beta.28.3`):

```
$ ng new angular-cli-playground
$ cd angular-cli-playground
$ npm install --save-dev angular-playground
$ ./node_modules/.bin/angular-playground
env: node\r: No such file or directory
```

The error seems to be gone when the following file is converted using `fromdos` tool (`brew install tofrodos`).

`fromdos ./node_modules/angular-playground/dist/bin/cli.js`

*Edit*

I'm running on macOS Sierra, node 7.5.0 with npm 4.1.2 . 

After doing the `fromdos` step I followed the guide and got it working. Will further explore the playground tomorrow! 👍

**Comment by jschwarty** [comment]
Thanks for the detailed info! Will bust out my Mac and try and reproduce this so I can get it solved. 😎

**Comment by jschwarty** [comment]
I think I tracked this down (thanks to your detailed info)! Working on a fix.

**Comment by jschwarty** [comment]
Fixed and published as **1.1.7**. See [changelog](https://github.com/SoCreate/angular-playground/blob/master/CHANGELOG.md) for details.

Thanks @beeman!

**Comment by beeman** [comment]
Awesome @jschwarty, thanks for the quick fix! Glad this issue has been useful 👍
